vcd_cli/catalog.py

```diff
@@ -114,6 +114,27 @@ def info(ctx, catalog_name, item_name):
     except Exception as e:
         stderr(e, ctx)
 
+
+@catalog.command(short_help='catalog control access details')
+@click.pass_context
+@click.argument('catalog-name',
+                metavar='<catalog-name>',
+                required=True)
+def control_access(ctx, catalog_name):
+    try:
+        client = ctx.obj['client']
+        org_name = ctx.obj['profiles'].get('org')
+        in_use_org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, in_use_org_href, org_name == 'System')
+        control_access = org.get_catalog_access_control_settings(catalog_name)
+        stdout('Access Settings for catalog :' + catalog_name)
+        stdout(control_access.get('AccessSettings'), ctx, show_id=True)
```

Comment:
None value for the control_access is not being currently handled in get_catalog_access_control_settings.
Line 131 - it may throw null pointer exception. Should we throw NPE or simply error out  gracefully with a msg "no access control settings yet"?

Reply:
Thanks for catching this one. I have made the None check and add a message if its None. Will update the pull request.